src/window/windows/castlewindow_winapi.inc

```diff
@@ -448,6 +453,10 @@ function TCastleWindowBase.WindowProc(uMsg: UINT; wParm: WPARAM; lParm: LPARAM):
     MaybeRelease;
   end;
 
+const
```

Comment:
These constants are part of `JwaDbt`. Since we use winunits-jedi anyway (`JwaWinUser` above), then we can use `JwaDbt` here too.